task/task-pangolin-pangoro/src/config.rs

```diff
@@ -146,6 +150,7 @@ impl ChainInfoConfig {
             port: host_port.2,
             signer: except_signer.or_else(|| self.signer.clone()),
             signer_password: self.signer_password.clone(),
+            transactions_mortality: Some(4),
```

Comment:
Why 4 here?

Reply:
4 might be too short, only 4 block time morality for extrinsics(24 s), and better to equal or less than 256, cause `BlockHashCountForCrab` is 256

https://github.com/darwinia-network/darwinia/blob/a965cd8d2c48e98644b0edbffe846f31cbfb3f6f/runtime/common/src/lib.rs#L67

Reply:
@WoeOm how polkadot js calculate this arg

Reply:
There is generate config template, we can put this document to template file. and there will be change to 256.

Reply:
Set default to 256 would be better.